contracts/stakingv2/modules/AdminSettings.sol

```diff
@@ -34,7 +74,7 @@ contract AdminSettings is StakingStateV2 {
         address _token,
         address _spender,
         uint256 _value
-    ) external onlyOwner {
+    ) public onlyOwner {
         IERC20(_token).safeApprove(_spender, _value);
```

Comment:
make sure to zero out the approval before calling this

Reply:
In most cases we will need infinit approval, lets just remember that we will need the second transaction if want to increase allowance from non zero value, only for USDT, just to save gas